packages/terra-notification-dialog/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra NotificationDialog
+
+[![NPM version](http://img.shields.io/npm/v/terra-notification-dialog.svg)](https://www.npmjs.org/package/terra-notification-dialog)
+[![Build Status](https://travis-ci.org/cerner/terra-framework.svg?branch=master)](https://travis-ci.org/cerner/terra-framework)
+
+`terra-notification-dialog` is a notification dialog component built over the `terra-abstract-modal`. It has the highest z-index of 8001. It is a common component to be used for confirmation/acceptance criteria style dialogs.
```

Comment:
*8000

Reply:
I think it's 8001.
https://github.com/cerner/terra-core/blob/master/packages/terra-abstract-modal/src/AbstractModal.scss

Reply:
Ahhh I missed that

Reply:
*9001 xD